std_msgs/Header header
string[] id
string[] state
string[] ip
string[] mac_address
